openmc/deplete/abc.py

```diff
@@ -76,14 +75,15 @@ def change_directory(output_dir):
         os.chdir(orig_dir)
 
 
-class TransportOperator(ABC):
-    """Abstract class defining a transport operator
+class DepletionOperator(ABC):
```

Comment:
I think `TransportOperator` more accurately describes what this series of classes is all about: given a composition, return k<sub>eff</sub> and reaction rates. Obviously the new `IndependentOperator` doesn't do any transport, but it also doesn't do any depletion. `ReactionRateOperator` could be more apt, although it doesn't reflect the k<sub>eff</sub> part...

Reply:
Okay, I'll switch it back to TransportOperator then.